Competition2017/tests/competition/subsystems/drive/commands/DriveToOrientationCommandTest.java

```diff
@@ -0,0 +1,86 @@
+package competition.subsystems.drive.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+import competition.subsystems.drive.DriveTestBase;
+import xbot.common.controls.actuators.MockCANTalon;
+
+public class DriveToOrientationCommandTest extends DriveTestBase {
+
+    DriveToOrientationCommand command;
+    
+    @Before
+    public void setup() {
+        
+        super.setUp();
+        
+        command = injector.getInstance(DriveToOrientationCommand.class);
+    }
+    
+    @Test
+    public void testTurnLeft90() {
+ 
+        setMockGyro(0);
+        command.setHeadings(90);
+ 
+        command.initialize();
+        command.execute();
+        
+        System.out.println("Left Power: " + ((MockCANTalon)drive.leftDrive).getSetpoint());
```

Comment:
It'd be preferable to remove these print calls so that we aren't putting more stuff into the logs than necessary.